rasa/validator.py

```diff
@@ -192,6 +192,29 @@ def verify_utterances_in_stories(self, ignore_warnings: bool = True) -> bool:
 
         return everything_is_alright
 
+    def verify_actions_in_stories_rules(self) -> bool:
+        """Verifies that actions used in stories and rules are present in the domain."""
+        visited = set()
+        everything_is_alright = True
+
+        for story in self.story_graph.story_steps:
+            for event in story.events:
+                if not isinstance(event, ActionExecuted):
+                    continue
+
+                if not event.action_name.startswith("action_"):
```

Comment:
why do we need this condition?

Reply:
@wochinge For two reasons:
1. there is already a method `verify_utterances_in_stories` that validates actions with `utter_` prefix in stories/rules
2. we were missing the validation of actions with `action_` prefix that should raise ActionNotFoundException based on what I understood from the issue thread

Do you think a different approach should be taken?

Reply:
Makes sense 👍🏻